contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -409,4 +461,26 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
             uint256(vaultState.lockedAmount).sub(unlockedAssetAmount)
         );
     }
+
+    /**
+     * @notice Settle USDC auction and swap the proceeds to underlying asset
+     * @param minAmountOut is the minimum amount of underlying acceptable for the swap
+     */
+    function settleAuctionAndSwap(uint256 minAmountOut)
+        external
+        onlyKeeper
+        nonReentrant
+    {
+        require(isUsdcAuction, "!isUsdcAuction");
+        require(minAmountOut > 0, "!_minAmountOut");
```

Comment:
Need to add test cases for these requires

Reply:
Added documentation